perf.env

```diff
@@ -81,5 +82,8 @@ CLEDG_DATABASE__PASSWORD=password
 CLEDG_DATABASE__SCHEMA=central_ledger
 
 ## MLAPI - ML API Adapter API/Handler Config
-MLAPI_TRANSFERS__SEND_TRANSFER_CONFIRMATION_TO_PAYEE=false
```

Comment:
Is there a reason you're removing this config?

Reply:
So I observed an issue when setting a non-string variable using env variable syntax like this. The service is failing to read the variable as boolen. Instead it is reading the variable as string which is not we expect. So all the non string variables should be set through config-modifier script instead of env variables.
Coming to this parameter `MLAPI_TRANSFERS__SEND_TRANSFER_CONFIRMATION_TO_PAYEE`, its already set through config-modifier. So removing from here.